suites/nautilus/rgw/tier-2_rgw_test-using-s3cmd.yaml

```diff
@@ -40,3 +40,13 @@ tests:
         script-name: ../s3cmd/test_bucket_stats.py
         config-file-name: ../../s3cmd/configs/test_bucket_stats.yaml
         timeout: 300
+
+  - test:
```

Comment:
The earlier name provided us an idea on what was being tested.. for example the feature or a functional area... could we add what is being tested using `s3cmd`

Reply:
Updated as per comment